src/components/pages/Notes/NoteGroupNav.js

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from 'react';
+import { connect } from "react-redux";
+import { values } from "lodash";
+import { NavLink, Route } from 'react-router-dom';
+
+import Button from "../../library/Button";
+
+
+function NoteGroupNav({ children, noteGroups, addGroup, deleteGroup, updateGroupName }) {
+  const [editedGroupId, setEditedGroupId] = useState();
+  const [editedGroupName, setEditedGroupName] = useState('jeanPaulBelmondo');
+
+  return (
+    <div className="group-nav">
+      <div className="columns">
+        <div className="column is-2">
+          <aside className="menu">
+            <ul className="menu-list">
+              {noteGroups.map(noteGroup => {
+                return(
+                    <li key={noteGroup.id}>
+                      {editedGroupId === noteGroup.id ? (
+                        <input
+                          className="input is-primary is-small"
+                          autoFocus
+                          value={editedGroupName}
+                          onChange={e => setEditedGroupName(e.target.value)}
+                        />
+                      ) : (
+                        <NavLink to={`/notes/noteGroups/${noteGroup.id}`} className="note-group">
+                          {noteGroup.name}
+                          <div className="actions">
+                            <Button
+                              additionalClassName="is-text edit-button"
```

Comment:
Rather than passing Bulma `is-text` class to the button, why not add a property `isText` to the button. It would be `false` by default so that you don't have to specify it each time.

Reply:
chenecompwendpas

Reply:
ben en gros c'est ajouter dans la classe Button:
```jsx
function Button({
  ...,
  isText
}) {

if (isText) className += ' is-text'
...
Button.defaultProps = {
  color: 'default',
  size: 'normal',
  isText: false
};
```

Puis tu utilises comme ça:
```jsx
<Button isText>Text button</Button>
```

Reply:
et bah voila, bcp plus clair comme ça mdrr